docs/how-to-use-the-solution/README.md

```diff
@@ -438,15 +441,15 @@ It is necessary to have a [scheduled meeting](https://support.microsoft.com/en-u
     Complete the body in Postman with the following: 
     ```json
     {  
-    "participantId": "{{participantId}}",  
-    "participantGraphId": "{{participantGraphId}}",  
-    "resourceType": {{resourceType}}, 
-    "protocol": {{protocol}}, 
-    "mode": {{mode}},
-    "streamUrl": "{{streamUrl}}",  
-    "streamKey": "{{streamKey}}", 
-    "timeOverlay": {{timeOverlay}},
-    "enableSsl" : {{enableSSl}}
+        "participantId": "{{participantId}}",  
+        "participantGraphId": "{{participantGraphId}}",  
+        "resourceType": "{{resourceType}}", 
```

Comment:
**suggestion (blocking)**: The problem with putting these placeholders in double quotes, is that these properties don't expect an string. By putting the double quotes here we are giving the impression the value should be an string.

Instead, lets put an example value and refer to the property name in reference table. E.g.:
```json
{
    "protocol": 1
}
```
| Placeholder | Description |
|-------------|-------------|
| protocol | Use `1` for RTMP extraction protocol |